projects/kerberos.cmake

```diff
@@ -10,17 +10,19 @@ set(KERBEROS_DOWNLOAD_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/kerberos)
 if(WIN32)
   set(KERBEROS_DL_URL http://web.mit.edu/kerberos/dist/kfw/4.0/kfw-4.0.1-src.zip)
   set(KERBEROS_DL_MD5 6d58ca865beb5b5c145dc4f579753d33)
+  set(KERBEROS_VER 4.0.1)
 else()
```

Comment:
Per http://web.mit.edu/kerberos/dist/index.html the current windows release is kfw 4.0.1...they have a different download entirely for windows vs linux.